bin/cideploy.sh

```diff
@@ -50,7 +50,7 @@ main() {
       cf auth $CF_USER $CF_PASSWORD
       cf target -o $CF_ORG
       cf target -s $CF_SPACE
-      cf push $appname
+      cf zero-downtime-push cgadocs -f manifest.yml --show-app-log
```

Comment:
Do we need to specify the appname `cgadocs` given it's in the manifest?

Reply:
Yes we do, the plugin gates doing a normal push vs a zero-downtime-deployment based on whether the app name is specified or not.  Maybe we should fix that on their end, but it's how it works for now.

Reply:
Actually, maybe we can remove from the manifest.yml.  Why don't I try that...